lib/adapter.js

```diff
@@ -375,7 +375,16 @@ module.exports = (function() {
 
         // Run Query
         client.query(query.query, query.values, function __CREATE__(err, result) {
-          if(err) return cb(err);
+          if (err) {
```

Comment:
We don't use createEach :( Also I'm pretty sure they do roughly the same thing ?